internal/pkg/agent/application/info/svc_windows.go

```diff
@@ -21,9 +23,10 @@ func RunningUnderSupervisor() bool {
 	if err != nil {
 		return false
 	}
-	defer windows.FreeSid(serviceSid)
+	defer func() { _ = windows.FreeSid(serviceSid) }()
 
-	t, err := windows.OpenCurrentProcessToken()
+	var t windows.Token
+	err = windows.OpenProcessToken(windows.CurrentProcess(), windows.TOKEN_QUERY, &t)
```

Comment:
unrelated change, fixing lint error: SA1019: windows.OpenCurrentProcessToken is deprecated: Explicitly call OpenProcessToken(CurrentProcess(), ...) with the desired access instead, or use GetCurrentProcessToken for a TOKEN_QUERY token. (staticcheck)